cmd/chisel/cmd_info_test.go

```diff
@@ -0,0 +1,250 @@
+`,
+}, {
+	summary: "Same package, multiple slices",
+	input:   infoRelease,
+	query:   []string{"mypkg_foo", "mypkg", "mypkg_baz"},
+	stdout: `
+package: mypkg
+archive: ubuntu
+slices:
+    bar:
+        essential:
+            - mypkg_foo
+        contents:
+            /bin/bar: {}
+            /etc/bar.conf: {text: TODO, mutable: true, arch: riscv64}
+            /lib/*-linux-*/bar.so: {arch: [amd64, arm64, i386]}
+            /usr/bin/bar: {symlink: /bin/bar}
+            /usr/bin/baz: {copy: /bin/bar}
+            /usr/lib/bar*.so: {}
+            /usr/share/bar/*.conf: {until: mutate}
+        mutate: |
+            dir = "/usr/share/bar/"
+            conf = [content.read(dir + path) for path in content.list(dir)]
+            content.write("/etc/bar.conf", "".join(conf))
+    baz:
+        essential:
+            - libpkg_libs
+            - mypkg_bar
+            - mypkg_foo
+    foo:
+        contents:
+            /etc/foo: {}
+            /etc/foo-dir/: {make: true, mode: 0644}
+`,
+}, {
+	summary: "Same slice, appearing multiple times",
+	input:   infoRelease,
+	query:   []string{"mypkg_foo", "mypkg_foo", "mypkg_foo"},
+	stdout: `
+package: mypkg
+archive: ubuntu
+slices:
+    foo:
+        contents:
+            /etc/foo: {}
+            /etc/foo-dir/: {make: true, mode: 0644}
+`,
+}, {
+	summary: "No slices found",
+	input:   infoRelease,
+	query:   []string{"foo", "bar_foo"},
+	err:     `no slice definitions found for: "foo", "bar_foo"`,
+}, {
+	summary: "Some slices found, others not found",
+	input:   infoRelease,
+	query:   []string{"foo", "mypkg_foo", "bar_foo"},
+	stdout: `
+package: mypkg
+archive: ubuntu
+slices:
+    foo:
+        contents:
+            /etc/foo: {}
+            /etc/foo-dir/: {make: true, mode: 0644}
+`,
+	err: `no slice definitions found for: "foo", "bar_foo"`,
+}, {
+	summary: "No args",
+	input:   infoRelease,
+	err:     "the required argument `<pkg|slice> (at least 1 argument)` was not provided",
+}, {
+	summary: "Empty, whitespace args",
+	input:   infoRelease,
+	query:   []string{"", "    "},
+	err:     `no slice definitions found for: "", "    "`,
+}, {
+	summary: "Bad format slices",
+	input:   infoRelease,
+	query:   []string{"foo_bar_foo", "a_b", "7_c", "a_b c", "a_b x_y"},
+	err:     `no slice definitions found for: "foo_bar_foo", "a_b", "7_c", "a_b c", "a_b x_y"`,
+}}
+
+var testKey = testutil.PGPKeys["key1"]
+var defaultChiselYaml = `
+	format: chisel-v1
+	archives:
+		ubuntu:
+			version: 22.04
+			components: [main, universe]
+			v1-public-keys: [test-key]
+	v1-public-keys:
+		test-key:
+			id: ` + testKey.ID + `
+			armor: |` + "\n" + testutil.PrefixEachLine(testKey.PubKeyArmor, "\t\t\t\t\t\t") + `
+`
+
+var infoRelease = map[string]string{
+	"chisel.yaml": string(defaultChiselYaml),
+	"slices/mypkg.yaml": `
+		package: mypkg
+
+		essential:
+			- mypkg_foo
+
+		slices:
+			foo:
+				contents:
+					/etc/foo:
+					/etc/foo-dir/: {make: true, mode: 0644}
```

Comment:
Now that we have set for the test style in the tests (see `slicer_test.go` and the test packages), let's change it to match that style and not a weird mix of invented files (foo, foo-dir, baz, bar, etc.) and real directories (/etc, /usr/bin, etc.)

Reply:
Done.